suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -45,24 +39,38 @@ const cameraStyle = prepareNativeStyle(utils => ({
     marginBottom: utils.spacings.medium,
 }));
 
-const xpubFormValidationSchema = yup.object({
-    xpubAddress: yup.string().required(),
-});
-type XpubFormValues = yup.InferType<typeof xpubFormValidationSchema>;
-
 export const XpubScanScreen = ({
     navigation,
     route,
 }: StackProps<AccountsImportStackParamList, AccountsImportStackRoutes.XpubScan>) => {
+    const { translate } = useTranslate();
     const { applyStyle } = useNativeStyles();
     const [_, setIsCameraRequested] = useState<boolean>(false);
     const { showAlert, hideAlert } = useAlert();
+
+    const { networkSymbol } = route.params;
+    const { networkType } = networks[networkSymbol];
+
+    const xpubFormValidationSchema = yup.object({
```

Comment:
Not great to have it inside the component. You can use yup context and pass the `networkSymbol` to the schema even when outside of the component.

Reply:
fixed in 8ec74f85a0